Cargo.toml

```diff
@@ -16,7 +16,8 @@ loggerv = "0.7.2"
 log = "0.4"
 clap = "2.33.3"
 regex = "1"
-procfs = "0.8.1"
+#procfs = "0.8.1"
+procfs = { git = "https://github.com/rossf7/procfs", branch = "cgroup-path-fix" }
```

Comment:
The systemd cgroup driver uses `:` as a separator but this gets truncated. So the final part with the container ID is missing.

This was fixed in https://github.com/eminence/procfs/pull/152 which was released in  0.11.0 but we can't upgrade due to https://github.com/hubblo-org/scaphandre/issues/69
 

```
12:memory:/system.slice/containerd.service/kubepods-burstable-
poda340e6e8_6f45_4b86_a694_5da042b07928.slice:cri-containerd:66848764f11844f8b3d92ee96365e0f35f279e7c7397efb684d394fb06bec2de
```